modules/medianetBidAdapter.js

```diff
@@ -24,15 +32,20 @@ function siteDetails(site) {
 }
 
 function getPageMeta() {
+  if (mnData.pageMeta) {
```

Comment:
only problem I see here is that this may not play well with concurrent auctions, but I since it looks like `mdData.pageMeta` would not change on a given page, it shouldn't be a problem.

Reply:
@vedantseta if you're ok with this, I'll merge.